tests/integration/dbapi/conftest.py

```diff
@@ -75,7 +75,8 @@ def all_types_query() -> str:
         'true as "boolean", '
         "[1,2,3,4] as \"array\", cast('1231232.123459999990457054844258706536' as "
         'decimal(38,30)) as "decimal", '
-        "cast(null as  int) as nullable"
+        "cast(null as  int) as nullable, "
+        "'abc123'::bytea as bytea"
```

Comment:
Just a heads-up, unquoted identifiers are not supposed to work and SQL team has just released a fix. We should change this to `as "bytea"` and any other new types that worked this way.